### ISSUE TITLE
Docs:save-a-one-to-one-relation error

### ISSUE THREAD

**Issue by gocpplua** [typeorm/typeorm#7844]
### What was unclear or otherwise insufficient?

[loading-from-the-database](https://typeorm.io/#undefined/save-a-one-to-one-relation)

when i run this code,this is a error:
```

> typeormdemo@1.0.0 start /data/gocpplua/gocpplua_node/mylearn/proj/typeorm
> ts-node src/index.ts

QueryFailedError: SQLITE_CONSTRAINT: NOT NULL constraint failed: photo.views
    at new QueryFailedError (/data/gocpplua/gocpplua_node/mylearn/proj/typeorm/src/error/QueryFailedError.ts:11:9)
    at Statement.handler (/data/gocpplua/gocpplua_node/mylearn/proj/typeorm/src/driver/sqlite/SqliteQueryRunner.ts:79:26) {
  errno: 19,
  code: 'SQLITE_CONSTRAINT',
  query: 'INSERT INTO "photo"("id", "name", "description", "filename", "views", "isPublished") VALUES (NULL, ?, ?, ?, NULL, ?)',
  parameters: [
    'Me and Bears',
    'I am near polar bears',
    'photo-with-bears.jpg',
    1
  ]
```

### Recommended Fix
i don't know how to solve it

- [ ] `aurora-data-api`
- [ ] `aurora-data-api-pg`
- [ ] `better-sqlite3`
- [ ] `cockroachdb`
- [ ] `cordova`
- [ ] `expo`
- [ ] `mongodb`
- [ ] `mysql`
- [ ] `nativescript`
- [ ] `oracle`
- [ ] `postgres`
- [ ] `react-native`
- [ ] `sap`
- [x] `sqlite`
- [ ] `sqlite-abstract`
- [ ] `sqljs`
- [ ] `sqlserver`

### Are you willing to resolve this issue by submitting a Pull Request?

- [ ] Yes, I have the time, and I know how to start.
- [x] Yes, I have the time, but I don't know how to start. I would need guidance.
- [ ] No, I don't have the time, although I believe I could do it if I had the time...
- [ ] No, I don't have the time and I wouldn't even know how to start.

**Comment by gocpplua** [comment]
i know how to resolve it

**Comment by pleerock** [comment]
fixed by #7845
